decidim-bulletin_board-app/app/assets/javascripts/sandbox/vote.es6

```diff
@@ -29,13 +30,16 @@ $(async () => {
   });
 
   await component.bindEvents({
-    onSetup() {},
+    onSetup() {
+      $castVote.removeAttr("disabled");
+    },
     onBindStartButton(onEventTriggered) {
       $castVote.on("click", onEventTriggered);
     },
     onStart() {},
     onVoteValidation(validVoteFn, invalidVoteFn) {
       $vote.css("background", "");
```

Comment:
same as before

Reply:
It's not the same as showing or hiding something. In this case, this is disabling the button, so it's not just aesthetics 😄